public/app/features/alerting/unified/Silences.tsx

```diff
@@ -50,6 +56,9 @@ const Silences: FC = () => {
 
   const getSilenceById = useCallback((id: string) => result && result.find((silence) => silence.id === id), [result]);
 
+  const mimirLazyInitError =
+    error?.message?.includes('the Alertmanager is not configured') && amFeatures?.lazyConfigInit;
```

Comment:
I understand this is the only possible conditional we can use for that so far. But I'm thinking if it could be possible that FE can check differently (maybe another BE endpoint) if this error is because mimir has no AM configured, instead of checking a text in the error message (something that could change meaning the same)

Reply:
Unfortunately, this is the only way to check if Mimir has no config. Maybe in the future, we'll have something better